xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -1555,14 +1555,25 @@ class ATTR_DLL_LOCAL CInstanceInputStream : public IAddonInstance
   //----------------------------------------------------------------------------
 
   //============================================================================
-  /// @brief Sets desired width / height
+  /// @brief Notify current screen resolution
   ///
   /// @param[in] width Width to set
   /// @param[in] height Height to set
   ///
   virtual void SetVideoResolution(int width, int height) {}
   //----------------------------------------------------------------------------
 
+  //============================================================================
+  /// @brief Notify current screen resolution and max screen resolution allowed
+  ///
+  /// @param[in] width Width to set
+  /// @param[in] height Height to set
+  /// @param[in] maxWidth Max width allowed
+  /// @param[in] maxHeight Max height allowed
+  ///
+  virtual void SetVideoResolution(int width, int height, int maxWidth, int maxHeight) {}
```

Comment:
As on other place defined as `unsigned int m_currentVideoWidth{0};` makes it maybe sense to use as `unsigned int` here.
If it can be never negative, better to use as `unsigned int` on all values.

Reply:
i can change to `unsigned int` but will be a breaking change for all addons
Is it still ok for you?

Reply:
change done